packages/office-ui-fabric-react/src/utilities/color/cssColor.ts

```diff
@@ -0,0 +1,104 @@
+import { IRGB } from './interfaces';
+import { MAX_COLOR_ALPHA } from './consts';
+import { COLOR_VALUES } from './colorValues';
```

Comment:
I wonder if we could just remove colorValues entirely... None of our code uses named colors as far as I can tell. Downside is that it could potentially break other consumers, but technically 1) this code was never part of our public API anyway (per @dzearing, only things importable from directly under `lib` are public) and 2) the capability of handling named colors was totally undocumented.

Reply:
Did you see my comment in the Fabric React Teams channel? I have a separate branch attempting to remove colorValues… but `getComputedStyle()` doesn't work well with named colors in jest tests.
We DO need the named colors since things like the color picker need to be able to pass it in. Swatch color picker also specifically tests using named colors.

Reply:
Oh sorry, didn't see that thread (found it now).